server/fetch-static-serving.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
Very good questions!
1. I have no idea whether this works on Windows! I'd in fact completely not thought about it! I don't have a Windows machine to test on, alas, and I have 0% knowledge about Windows development - I haven't even used a Windows box since XP... 😮 Do you have the possibility to try it and/or suggest a cross-platform approach?
2. Bazel intuitively does seem like overkill, though it would indeed solve the problem.
3. Localization etc. remains the exact same story as before: the script generates every YAML file in every localization, we continue to upload the same set of ~2000 localized YAML files.

Reply:
I've filed an issue.